### PR TITLE
docs(guide): convert /tutorials/commands.md to rst

### PR DESCRIPTION
**Pull request by daprahamian**
Fixes NODE-2198

## Description

**What changed?**

**Are there any files to ignore?**

### CODE REVIEW

File: docs/guide/tutorials/commands.txt [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+Database commands allow you to perform a wide range of diagnostic and administrative
+tasks with the Node.js driver. For example, the
+:manual:`dbStats </reference/command/dbStats/>` command returns
+storage statistics for a given database. Use the ``command`` function to access
+database commands.
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+   const assert = require('assert');
```
Comment (daprahamian) [inline docs/guide/tutorials/commands.txt]: assert isn't used here. We should get rid of it

Comment (mbroadst) [reply docs/guide/tutorials/commands.txt]: agreed

File: docs/guide/tutorials/commands.txt [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+   const url = 'mongodb://localhost:27017';
+   
+   // Database Name
+   const dbName = 'myproject';
+   
+   // Create a new MongoClient
+   const client = new MongoClient(url);
+   
+   // Use connect method to connect to the Server
+   client.connect().then(async function() {
```
Comment (mongomoe) [inline docs/guide/tutorials/commands.txt]: `   client.connect().then(async function() {`

 should be es6 arrow fn:

 `client.connect().then(async () => {`

**Comment by daprahamian** [comment]
Updated this one. IMO the boilerplate for this one makes it hard to see what is going on. How do people feel about doing one of the following:

1. Only including the main function here:

    ```js
    async function main(client) {
      const db = client.db('myproject');
      const results = await db.command({ dbStats: 1 });
      console.log(results);
    }
    ```
2. Using the `:emphasize-lines:` option on the `code-block` to emphasize the lines with the main fn?

    ```rst
    .. code-block:: js
      :emphasize-lines: 9-13

      const { MongoClient } = require('mongodb');
      
      // Connection URL
      const url = 'mongodb://localhost:27017';
      
      // Create a new MongoClient
      const client = new MongoClient(url);
      
      async function main(client) {
        const db = client.db('myproject');
        const results = await db.command({ dbStats: 1 });
        console.log(results);
      }

      // Function to connect to the server and run your code
      async function run() {
        try {
          // Connect the client to the server
          await client.connect();
          console.log('Connected successfully to server');

          await main(client);
        } finally {
          // Ensures that the client will close when you finish/error
          await client.close();
        }
      }

      // Runs your code
      run();
    ```

**Comment by daprahamian** [comment]
screenshot of what it looks like with emphasize-lines:

Screen Shot 2019-10-17 at 6 19 33 PM

**Pull request merged**